### ISSUE TITLE
Get steempress markdown processing to sync with steemit

### ISSUE THREAD

**Issue by bmann** [aaroncox/steempress#4]
e.g. the #-tagged items and @-mentions of user accounts in the body should link to either steemit or local blog (a config setting).

**Comment by aaroncox** [comment]
For reference, the code that handles the parsing of content currently is located here:

https://github.com/aaroncox/steempress/blob/master/src/SteemPress/Model/Comment.php#L29-L41

**Comment by bmann** [comment]
Good stuff. I'm going to just keep filing issues, and some of them I'll take care of myself :)

I also got @ntomaino up and running and got a steemithunt instance up so we have some "real" sites up. We'll centralize issues here.

**Comment by aaroncox** [comment]
Absolutely, it'll be a good way for us to keep track of what needs to happen.
